DistFiles/factoryCollections/Templates/Wall Calendar/Wall Calendar.jade

```diff
@@ -22,6 +22,8 @@ html
 		meta(name='defaultBookletLayout', content='Calendar')
 		meta(name='pageTemplateSource', content='Wall Calendar')
 		meta(name='experimental', content='true')
+		//- only added in version 3.3
+		meta(name='lockedDownAsShell', content='true')
```

Comment:
Locked down as shell changes other behaviors too...for example, it removes at least one tab of the format dialog. Have you looked into what all of them are and confirmed it's OK to do all of them? A shell is usually a book with text to be translated, so marking one of our book-creationg templates as a shell feels wrong (code getting worse :-). @hatton, what do you think? Should we just invent a new meta flag for "templateWithFixedPageSet"?
Of course we also have a page flag for singletons, but I guess we can't use that here because the content pages, while having a predetermined number of instances, occur more than once. They're also not really xmatter, though marking them so would also probably fix the problem.